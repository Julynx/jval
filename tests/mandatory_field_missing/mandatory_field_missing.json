{
    "role": "user",
    "cars": [
        {
            "type": "car",
            "brand": "Toyota",
            "model": "Corolla",
            "year": 2020
        }
    ]
}