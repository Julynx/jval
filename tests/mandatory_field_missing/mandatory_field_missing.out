Validation error at 'name': missing value for key 'name'